instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-resteasy-3.0/javaagent/build.gradle.kts

```diff
@@ -43,6 +43,7 @@ dependencies {
   testLibrary("io.undertow:undertow-servlet:1.4.28.Final")
   testLibrary("org.jboss.resteasy:resteasy-servlet-initializer:3.0.4.Final")
 
+  latestDepTestLibrary("org.jboss.resteasy:resteasy-servlet-initializer:5.+")
```

Comment:
perhaps it would be better to use `3.+`here to align with the version used for the rest of the dependencies

Reply:
I just assumed that before we clearly used some 5.x version